### ISSUE TITLE
Remove await

### ISSUE THREAD

**Issue by flowchartsman** [johnalanwoods/maintained-modern-unix#5]
It [doesn't look like](https://github.com/slavaGanzin/await/issues/2) await is going to be packaged in any appreciable way any time soon, and release artifacts for other architectures [don't appear](https://github.com/slavaGanzin/await/issues/6) to be a priority. No judgement there, but this makes the app somewhat inaccessible, and another alternative might be better, such as [Rush](https://github.com/shenwei356/rush).

**Comment by johnalanwoods** [comment]
Thanks, will review

**Comment by johnalanwoods** [comment]
I'm on the fence here, any others feel strongly?

**Comment by johnalanwoods** [comment]
Inclined to agree, especially now we have Doggo - I'll update.

**Comment by kaddkaka** [comment]
@yozachar why, are there any problems with them?

Regarding await: Does it need package to be usable/fit on this list? I would vote to keep it.

**Comment by yozachar** [comment]
> I think, unmaintained tools can be demoted.

By `demote` I mean, to have another table at the end of the main table, which contains tools that have not been committed to, for over a year.

> @yozachar why, are there any problems with them?

If you're asking why aren't they being maintained, I do not know. If no commits are being made, for over an extended period (say 3 years), the tool is very likely abandoned/replaced. After all, this list is meant to hold "maintained" modern *nix tools

> Regarding await: Does it need package to be usable/fit on this list?

No, but preferably maintained.

**Comment by kaddkaka** [comment]
> > I think, unmaintained tools can be demoted.
> 
> By `demote` I mean, to have another table at the end of the main table, which contains tools that have not been committed to, for over a year.
> 
It could be a problem, but the tool could also be "done". But "demote" is preferable over removal.

> > @yozachar why, are there any problems with them?
> 
> If you're asking why aren't they being maintained, I do not know. If no commits are being made, for over an extended period (say 3 years), the tool is very likely abandoned/replaced. After all, this list is meant to hold "maintained" modern *nix tools
> 
I think the "maintained" name part just reflects that *this* repo is maintained, as opposed to the one that was forked. Not that the listed tools are maintained. (correct me if I'm wrong)

**Comment by johnalanwoods** [comment]
I hear you all on this.

Suggest we remove Dog, and watch the other two for now.

If all agreed I'll make the changes.

**Comment by johnalanwoods** [comment]
Addressed by https://github.com/johnalanwoods/maintained-modern-unix/pull/23

**Comment by slavaGanzin** [comment]
Hi there. Await author here. 

Couple points: 
1. There is no feature requests for await, and it works for my use cases.  So there is nothing to maintain
2. I forgot to add Mac version to releases, will spend sometime on proper CI. That’s why it 0.999, and not 1.0
3. Await is an inverse of rush/parallel: it runs tasks in parallel and not running single task with different inputs in parallel

**Comment by yozachar** [comment]
Hi @slavaGanzin, thank you for chiming in!

In #23 we kept `await` on `README.md`. I suppose there would be some commit activity over a span of 3 (or at least within 5) years, on the repository. But I also understand that if a program is "complete", then 'modifying' it doesn't make sense. The last commit date is not perfect, but a good indicator to check if program has been abandoned or not.

Well, anybody is welcome, to suggest a strategic (but not complicated) way to distinguish between complete and abandoned programs.

**Comment by mh166** [comment]
While, there's probably not that *one* great solution. Personally, I look at the last commit time, too, but then I also look at the number of open issues and PRs.

- For `await`, for example, it's only three PRs that aren't bug reports. That tells me that, all in all, people seem to be happy with the state of the program.

It would be a different thing if the last commit is 1+ years ago and there are over a dozen open bug reports with no comments by the author. This would more strongly indicate to me that I'm looking at an abandoned project.

- See `dog` for example: 12 open PRs with content reaching from simple typo fixes, updating deps to adding features. None of which have a comment of the maintainer. And 61 open issues ranging from feature requests to bug reports and also questions about the projects current status.


**TL;DR:** while the commit date is a first indicator, also look at the open Issues and PRs. See if they are about breaking bugs or asking about the project's status. And see if the maintainer(s) are chiming in with comments.

**Comment by seraphyn** [comment]
@mh166 Sign this.
await is definitively not abandoned

**Comment by slavaGanzin** [comment]
Updated release with MacOs version. Added additional installation ways.
